core/templates/dev/head/pages/admin/admin.html

```diff
@@ -3,7 +3,8 @@
   <head>
     <title>Oppia Admin Panel</title>
 
-    {% include 'pages/header_css_libs.html' %}
+    <!-- Require head styles file using webpack -->
+    @require('../header_css_libs.html')
```

Comment:
Can we use absolute path here? `pages/header_css_libs.html`

Reply:
I tried this, but webpack isn't able to find the file.
the require loads the file based on its current location

Reply:
Could you look into it more? Why webpack accepts absolute paths in `.ts` files but not HTML files?

Reply:
I don't think the problem is from webpack. I think it's from the underscore lodash template loader

Reply:
Maybe there is some way to config it for the loader?

Reply:
Please create an issue about this so we can resolve it later.

Reply:
Yes, but the loader being used does not support this yet.
One way we can do it would be to add the absolute path url as a webpack option and then use it to concatenate the string in the require.

Reply:
@vojtechjelinek I have created the issue here #6847